### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by nickolashe**

### CODE REVIEW

File: openCurrents/views.py [hunk 1]
```diff
@@ -3588,6 +3570,7 @@ def sendContactEmail(template_name, template_content, merge_vars, admin_email, u
 def sendTransactionalEmail(template_name, template_content, merge_vars, recipient_email, **kwargs):
 
     # adding launch function marker to session for testing purpose
+    test_time_tracker_mode = None
```
Comment (nickolashe) [inline openCurrents/views.py]: @tevtonez uninitialized `test_time_tracker_mode` was causing a runtime crash for sending most emails on development server

File: openCurrents/views.py [hunk 2]
```diff
@@ -3613,11 +3596,12 @@ def sendTransactionalEmail(template_name, template_content, merge_vars, recipien
             message=message
         )
     else:
-        print "We don't send Transactional emails during tests."
+        logger.debug('test mode: mocking mandrill call')
 
 def sendBulkEmail(template_name, template_content, merge_vars, recipient_email, sender_email, **kwargs):
 
     # adding launch function marker to session for testing purpose
+    test_mode = None
```
Comment (nickolashe) [inline openCurrents/views.py]: same as above, was failing to send bulk emails. probably related to the problem with invite volunteer emails as well.

**Comment by nickolashe** [comment]
@tevtonez still some failures in unit tests

**Comment by nickolashe** [comment]
more importantly, some runtime errors as well (lookup errors). lets address these at high priority.

**Comment by tevtonez** [comment]
@nickolashe thanks, the fixes for the failing tests are in PR #881

**Pull request merged**